Jenkinsfile

```diff
@@ -1,99 +1,5 @@
-                                -Dsonar.login=${SONAR_TOKEN} \
-                                ")
-                        }
-                    }
-                }
-            }
-        }
-        stage('Push artifacts') {
-            when {
-                expression { return (env.BRANCH_NAME in dockerTags || env.TAG_NAME ) }
-            }
-            steps {
-                script {
-                    if (env.BRANCH_NAME in dockerTags) {
-                        withCredentials([usernamePassword(credentialsId: 'bot-soramitsu-rw', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://docker.soramitsu.co.jp"
-                            env.TAG = dockerTags[env.BRANCH_NAME]
-                            sh "./gradlew dockerPush"
-                        }
-                        withCredentials([usernamePassword(credentialsId: 'nexus-nbc-deploy', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19000"
-                            env.TAG = dockerTags[env.BRANCH_NAME]
-                            sh "./gradlew dockerPush"
-                        }
-                    } else if (env.TAG_NAME ==~ /^bakong-\d{1,4}\.\d.*/) {
-                        def tagPattern = (env.TAG_NAME =~ /(?<=bakong-)(\d{1,4}\.\d.*)/)[0][1]
-                        println "${tagPattern}"
-                        withCredentials([usernamePassword(credentialsId: 'nexus-nbc-deploy', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://nexus.iroha.tech:19000"
-                            env.TAG = "${tagPattern}"
-                            sh "./gradlew dockerPush"
-                        }
-                    } else if (env.TAG_NAME ==~ /^\d.*/) {
-                        withCredentials([usernamePassword(credentialsId: 'bot-soramitsu-rw', usernameVariable: 'DOCKER_REGISTRY_USERNAME', passwordVariable: 'DOCKER_REGISTRY_PASSWORD')]) {
-                            env.DOCKER_REGISTRY_URL = "https://docker.soramitsu.co.jp"
-                            env.TAG = env.TAG_NAME
-                            sh "./gradlew dockerPush"
-                        }
-                    } else {
-                        error "Your git tag doesn't match any pattern!"
-                    }
-                }
-            }
-        }
-    }
-    post {
-        always {
-            publishHTML (target: [
-                allowMissing: false,
-                alwaysLinkToLastBuild: false,
-                keepAll: true,
-                reportDir: 'build/reports',
-                reportFiles: 'd3-test-report.html',
-                reportName: "D3 test report"
-            ])
```

Comment:
Do we need this report?

Reply:
I will return it back

Reply:
@Mingela , told what they did not use it. 
We will configure universal junit xml reports for all repos later.